src/util/image/MapCreator.java

```diff
@@ -544,7 +554,20 @@ private static String getValue(final String arg) {
 
   private static void handleCommandLineArgs(final String[] args) {
     final String[] properties = getProperties();
-    if (args.length == 1) {
+    boolean laf = false;
+    if(args.length == 1){
+      try {
+        UIManager.setLookAndFeel(args[0]);
+        laf = true;
+      } catch (ClassNotFoundException | InstantiationException | IllegalAccessException
+          | UnsupportedLookAndFeelException e) {
+        ClientLogger.logError("Look and feel could not be applied to the Map Creator", e);
+      } catch(ClassCastException e){
+        ClientLogger.logQuietly("Argument is not a Look and Feel");
+      }
+    }
+    
+    if (args.length == 1 && !laf) {
       String value;
       if (args[0].startsWith(TRIPLEA_MAP_FOLDER)) {
```

Comment:
Yeah, this is getting dirty.. If the first arg is a valid look and feel class, we use it, otherwise we assume it to be a folder location?? Args should probably have flags for them so it's not dependent upon arg position and value.